lsp/svelte.lua

```diff
@@ -21,6 +21,14 @@ return {
     end
   end,
   on_attach = function(client, bufnr)
+    -- Workaround to trigger reloading JS/TS files
+    -- See https://github.com/sveltejs/language-tools/issues/2008
+    vim.api.nvim_create_autocmd('BufWritePost', {
+      pattern = { '*.js', '*.ts' },
+      callback = function(ctx)
+        client:notify('$/onDidChangeTsOrJsFile', { uri = ctx.match })
```

Comment:
the context that @jasonlyu123 provided would be useful here, or some other reference
```suggestion
        -- internal API to sync changes that have not yet been saved to the file system
        client:notify('$/onDidChangeTsOrJsFile', { uri = ctx.match })
```